builds/jazz-build-module/scm-module.groovy

```diff
@@ -338,4 +338,27 @@ def getRepoCommitterInfo(commitHash) {
     }
 }
 
+def updateAclPermission(serviceId, policiesList, auth_token, aclUrl) {
```

Comment:
This can't be in SCM module (since we are actually calling ACL APIs here and not interacting with SCM anymore). Can you create a separate module for acl related util functions? May be, `acl-module.groovy`?

Reply:
Created `acl-module.groovy` and moved respective methods to this file.